pkg/driver/camera/camera_windows.cpp

```diff
@@ -347,6 +359,7 @@ int openCamera(camera* cam, const char** errstr)
   if (src == nullptr || dst == nullptr ||
       FAILED(hr = graphBuilder->Connect(src, dst)))
   {
+    printErr(hr);
```

Comment:
does the error need to be printed here?

Reply:
It's just printing error for debug, I will resolve it